### PR TITLE
Junit report -  add required fields

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java [hunk 1]
```diff
@@ -1874,8 +1874,15 @@ private void writeJunitReport( BuildJob buildJob, String safeFileName, Execution
     {
         File reportFile = new File( reportsDirectory, "TEST-" + safeFileName + ".xml" );
         Xpp3Dom testsuite = new Xpp3Dom( "testsuite" );
-        testsuite.setAttribute( "tests", "1" );
+        testsuite.setAttribute("name", junitPackageName + "." + safeFileName);
```
Comment (Tibor17) [inline src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java]: pls follow the checkstyle as it is everywhere: spaces, thx
Most probably you did not run the build (mvn -P run-its verify) on this project, check for errors, thx.

**Comment by Tibor17** [comment]
@eolivelli 
I don't know if invoker's `TEST-project.xml` has the attributes `errors`, `skipped`, `failures`. I have never seen it before. So I cannot fully evaluate the fix.

File: src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java (hunk 1 above)
Comment (olamy) [inline src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java]: testcase has the test name. This one should be only the package name as it's a suite?

File: src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java [hunk 2]
```diff
@@ -1874,8 +1874,15 @@ private void writeJunitReport( BuildJob buildJob, String safeFileName, Execution
     {
         File reportFile = new File( reportsDirectory, "TEST-" + safeFileName + ".xml" );
         Xpp3Dom testsuite = new Xpp3Dom( "testsuite" );
-        testsuite.setAttribute( "tests", "1" );
```
[the remark this answers is not in the document]
Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java]: it was moved, together with other attributes - see line 1881

File: src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java (hunk 1 above)
Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java]: it is like in maven-surefire plugin, it is result after test with sonarqube - this is required to be set to something

**Comment by slawekjaranowski** [comment]
I got testsuite attributes from 
https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd

File: src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java (hunk 1 above)
Comment (slawekjaranowski) [reply src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java]: fixed

File: src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (olamy) [reply src/main/java/org/apache/maven/plugins/invoker/AbstractInvokerMojo.java]: ooops sorry I didn't catch that

**Pull request merged**